### PR TITLE
fix(Pipelines): generate random string to avoid pod names collision

### PR DESCRIPTION
**Pull request by cheikhgwane**
Fix for issue about[ pipeline pod names](https://github.com/BLSQ/openhexa-team/issues/530).

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Append random string to pipeline pod name.

### CODE REVIEW

**Review by pvanliefland — CHANGES_REQUESTED** [review]
@cheikhgwane see my comments

File: hexa/pipelines/management/commands/pipelines_runner.py [hunk 1]
```diff
@@ -38,6 +39,7 @@ def run_pipeline_kube(run: PipelineRun, env_var: dict):
         ]  # Max length of a pod name is 63 characters
         + "-"
         + exec_time_str
+        + get_random_string(15)
```
[the remark this answers is not in the document]
Comment (cheikhgwane) [reply hexa/pipelines/management/commands/pipelines_runner.py:42]: Oh yes, you right.

**Review by qgerome — APPROVED** [review]

**Comment by qgerome** [comment]
LGTM!

**Review by pvanliefland — CHANGES_REQUESTED** [review]
I think it's good as it is but I would like to see a test for this one. It would probably imply moving the name generation logic in a separate method.

File: hexa/pipelines/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from django.utils.crypto import get_random_string
+
+from hexa.core.test import TestCase
+from hexa.pipelines.utils import generate_pipeline_container_name
+
+
+class PipelineUtilsTest(TestCase):
+    def test_generate_pipeline_container_name(self):
+        run = MagicMock()
+        run.code = get_random_string(50)
```
Comment (pvanliefland) [inline hexa/pipelines/tests/test_utils.py:12]: Let's test with 100 characters as well

**Review by pvanliefland — CHANGES_REQUESTED** [review]

**Review by pvanliefland — APPROVED** [review]
LGTM!

**Pull request merged**